src/static/js/modules/youtube.js

```diff
@@ -7,8 +7,6 @@
 
```

Comment:
I think we will need two files. One for VideoPlayer and the other for YouTube, which will extend the VideoPlayer.